pkg/install/hypershift.go

```diff
@@ -146,20 +148,30 @@ func (c *UpgradeController) RunHypershiftCleanup(ctx context.Context) error {
 }
 
 // This is run when:
-//   1) the controller starts up
-//   2) when the hypershift installation options (secrets and configmap) change
-func (c *UpgradeController) RunHypershiftInstall(ctx context.Context) error {
-	c.log.Info("enter runHypershiftInstall")
+//   1) the controller starts up, controllerStartup = true
+//   2) when the hypershift installation options (secrets and configmap) change, controllerStartup = false
+func (c *UpgradeController) RunHypershiftInstall(ctx context.Context, controllerStartup bool) error {
```

Comment:
nit.  Would you consider wrapping both calls to this func with better names??  Or update the comments before you call it with what "true" means?

Maybe RunHypershiftInstallOnStartup  and RunHypershiftUpdate  and apply the true/false in the func call to RunHypershiftInstall